src/parser/ts-for-v5/resolve-project-list.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Code from: https://github.com/typescript-eslint/typescript-eslint/blob/v5.62.0/packages/typescript-estree/src/parseSettings/resolveProjectList.ts
+ * Code updated to exclude caching.
+ */
+
+import { sync as globSync } from "globby";
+import isGlob from "is-glob";
+import * as path from "path";
+import * as ts from "typescript";
```

Comment:
Do you think users who don't have typescript installed will get an error on import?

We may need to load it dynamically inline instead.

`createRequire(path.join(process.cwd(), "__placeholder__.js"))("typescript")`

Reply:
Good catch, thank you! However, I have one question. What is `"__placeholder__.js"`?

Reply:
`"__placeholder__.js"` has no particular meaning. It is okay to use `index.js` or `foo.js` 😉